### ISSUE TITLE
Term Request: Dandruff-like corneal stromal flecks

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11747]
**Preferred term label:**
Dandruff-like corneal stromal flecks

**Synonyms**
* Corneal stromal flecks
* Mouchetée corneal stromal opacities
* Small discoid corneal stromal opacities

**Definition (free text, please give PubMed ID)**
Small, flat, discrete, gray-white, dandruff-like, or ring-shaped opacities scattered sparsely throughout all depths and sectors of an otherwise completely clear corneal stroma up to the limbus. Histologically, they correspond to swollen, vacuolated keratocytes containing glycosaminoglycans and complex lipids (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Opacification of the corneal stroma (HP:0007759)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Fleck corneal dystrophy (FCD) (ORPHA:98970, OMIM:121850)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
